test/smoke/src/areas/positron/notebook/notebookCreate.test.ts

```diff
@@ -39,10 +39,8 @@ export function setup(logger: Logger) {
 
 				await app.workbench.positronNotebooks.selectInterpreter('Python Environments', process.env.POSITRON_PY_VER_SEL!);
 
-				await expect(async () => {
```

Comment:
Removed the toPass wrapper because when the retry fired, it would add more code to the cell so that it would end up like:
eval("8**2")eval("8**2")
Need to find a better approach to retries here if occasional failures pop up.